src/main/java/org/gridsuite/loadflow/server/service/LoadFlowResultContext.java

```diff
@@ -66,11 +66,11 @@ public static LoadFlowResultContext fromMessage(Message<String> message, ObjectM
     }
 
     @Override
-    protected Map<String, String> getSpecificMsgHeaders() {
+    protected Map<String, Object> getSpecificMsgHeaders() {
```

Comment:
Why using Map<String, Object> instead of Map<String, String> (from AbstractResultContext)

Reply:
Leftover of the voltage init parsing mixup. Reverted. And ObjectMapper added.